signup.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
+    <title>login</title>
+</head>
+<body>
+    <div class="logo_box">
+        <img class="logo" src="images/login/logo.png">
+        <div class="login_box">
+            <div id="email">
+                <span class="email">이메일</span>
+                <h2 class="email_input"></h2>
```

Comment:
❗️ 수정요청
input 태그로 작성하셔서 유저를 통해 값을 받을 수 있게 해주세요~

```suggestion
                <input class="email_input" type="text" />
```